web/template/edit_payment.gohtml

```diff
@@ -50,9 +50,9 @@
 
                 <a href="#" class="govuk-link govuk-link--no-visited-state govuk-!-display-inline-block" data-module="select-todays-date">Use today's date</a>
 
-                <div class="govuk-button-group">
+                <div class="govuk-button-group govuk-!-padding-top-6">
                     <button class="govuk-button" data-module="govuk-button" type="submit">Save</button>
-                    <a data-app-iframe-cancel class="govuk-link govuk-link--no-visited-state" href="#">Cancel</a>
+                    <a class="govuk-link govuk-link--no-visited-state" href="{{ prefix (printf "/payments?id=%d" .Case.ID) }}">Cancel</a>
```

Comment:
Cancel can now be used to return back to the all payments page, (discussed and confirmed with Katie & Claire) otherwise users would have to go back out to actions and select the 'fees' button to get back to that screen